packages/@aws-cdk/aws-glue/test/security-configuration.test.ts

```diff
@@ -17,7 +16,6 @@ test('a security configuration with encryption configuration requiring kms key a
   const key = kms.Key.fromKeyArn(stack, 'ImportedKey', keyArn);
 
   const securityConfiguration = new glue.SecurityConfiguration(stack, 'SecurityConfiguration', {
-    securityConfigurationName: 'name',
```

Comment:
could we have a test for each of these (security configuration, database, and table) that verifies that the name parameter is still used when passed in?